tutorials-apps/bulk-question-delete-app/jest.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  setupFilesAfterEnv: ['<rootDir>/setUpTests.js'],
+  "moduleNameMapper": {
+    "\\.svg": "<rootDir>/__mocks__/svgrMock.js",
```

Comment:
@janarthananS1729 we are not testing this app right? i think we can avoid these config files